### PR TITLE
chore: Temp PR to resolve conflicts

### PR DESCRIPTION
**Pull request by hjetpoluru**
## **Description**

This is temp PR to practice resolve conflicts in the RC v12.0.0

[Open in GitHub Codespaces](https://codespaces.new/MetaMask/metamask-extension/pull/25527?quickstart=1)

### CODE REVIEW

File: test/e2e/fixture-builder.js [hunk 1]
```diff
@@ -10,10 +10,233 @@ const { CHAIN_IDS, NETWORK_TYPES } = require('../../shared/constants/network');
 const { SMART_CONTRACTS } = require('./seeder/smart-contracts');
 const { DAPP_URL, DAPP_ONE_URL } = require('./helpers');
 const { DEFAULT_FIXTURE_ACCOUNT, ERC_4337_ACCOUNT } = require('./constants');
-const {
-  defaultFixture,
-  FIXTURE_STATE_METADATA_VERSION,
-} = require('./default-fixture');
+
```
Comment (danjm) [inline test/e2e/fixture-builder.js:13]: Most of these changes are not in https://github.com/MetaMask/metamask-extension/pull/25426 or https://github.com/MetaMask/metamask-extension/pull/25426 and so most of these should not be here

File: ui/pages/routes/routes.component.test.js [hunk 2]
```diff
@@ -92,6 +92,73 @@ describe('Routes Component', () => {
   });
 
   describe('render during send flow', () => {
+    it('should render with network change disabled while adding recipient for send flow', async () => {
```
Comment (danjm) [inline ui/pages/routes/routes.component.test.js:95]: Most of these changes are not in https://github.com/MetaMask/metamask-extension/pull/25426 or https://github.com/MetaMask/metamask-extension/pull/25426 and so most of these should not be here

Comment (hjetpoluru) [reply ui/pages/routes/routes.component.test.js:95]: I agree that the code difference should highlight the changes, but I am unsure how to do so through IDE used(Visual Studio Code).
Screenshot 2024-06-26 at 2 24 10 PM
Screenshot 2024-06-26 at 2 24 54 PM

File: test/e2e/fixture-builder.js (hunk 1 above)
Comment (danjm) [reply test/e2e/fixture-builder.js:13]: and this seems to remove the usage of the `defaultFixture`, which we should not do

**Comment by danjm** [comment]
closing in favour of https://github.com/MetaMask/metamask-extension/pull/25548

**Pull request closed without merging**